### ISSUE TITLE
[Feature] Better documentation for the upgrade path when using Docker

### ISSUE THREAD

**Issue by MightyCreak** [FreshRSS/FreshRSS#4942]
**Is your feature request related to a problem? Please describe.**
I had an instance of FreshRSS 1.19.2 and I wanted to upgrade to FreshRSS 1.20.2 and it went bad... like, very bad.

I did try to backup and upgrade the containers volumes and everything, but I just kept having connection issues with the DB (I'm using PostgreSQL). In the end, I wiped out everything and set up a brand new instances. I did export the OPML, but I lost my user config (and all the data of the other users that were on my instance). It's not a commercial instance, so it's not dramatic. But I've seen way better upgrade processes. Especially, using Docker should ease these processes in the end, not hinder them.

The fact that PostgreSQL usage is somehow hidden in the documentation doesn't really help. The Docker documentation isn't easy to understand, there are a lot of stuff in it and, as a mere admin (not a dev or anything), you just want to know how to install and how to upgrade, basically.

**Describe the solution you’d like**
Two folds:
* Have a clear upgrade path in the documentation, for each release. Even if it's to say you don't need to do anything
* Simpler Docker documentation for when you just want to set up an instance or upgrade one

It would also be nice not to use a composition of `docker-compose.yml` files. I know it's a thing, but oh boy! it is cumbersome. I know the developer in me like the factorization that it provides, but as an admin, I'd rather have two completely separate files, like `docker-compose.sqlite.yml` and `docker-compose.postgres.yml`.

**Describe alternatives you’ve considered**
I did try to "do it right" backuping my volumes in order to restore them on the new containers, but it went bad pretty fast. In the end it was just too much trouble and I decided to wipe everything.

**Additional context**
I do know Docker pretty well, but I'm not a SysAdmin either.

**Comment by Alkarex** [comment]
The point is that there is absolutely nothing to do for such an upgrade except changing from `freshrss/freshrss:1.19.2` to `freshrss/freshrss:1.20.2`, or just stick to `freshrss/freshrss` or `freshrss/freshrss:edge`, and then run a `docker-compose ... pull` and `docker-compose ... up` as documented.
Could you try to explain a bit more what you have done?
Using PostgreSQL, you must however be careful to stick to a given PostgreSQL major version, or follow PostgreSQL's update instructions.

**Comment by MightyCreak** [comment]
Yep, I guess my mistake was to upgrade the Postgres image as well 😕

I was on `postgres:12-alpine` and the new compose-docker.yml is at `postgres:14`. But despite that, I was quite confused with the new `docker-compose.yml` files that you had to compose. Also the variable names changed as it was `POSTGRES_*` in the `.env` and it was changed to `DB_*`. I completely understand why, I just think it would be nice to help those who will upgrade FreshRSS later, even if it is fairly easy, knowing that it is can change how you anticipate the upgrade (and knowing that we can't easily change postgres release can help too :sweat_smile: )
